src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.cpp

```diff
@@ -37,18 +37,126 @@ tnsr::aa<DataType, Dim, Fr> compute_spacetime_metric(
   return spacetime_metric;
 }
 
+template <size_t Dim, typename Fr, typename DataType>
```

Comment:
Fr is used for the frame template parameter in Tensor and other places.  Should that be changed too?  (I agree with Dim->SpatialDim since otherwise its confusing for spacetime quantities)

Reply:
Probably, it's a vestige of when Frame was an enum class instead of a collection of structs so we had to type 'Frame Fr' as the template parameter.